projects/npm-tools/packages/npm-scripts/test/sass/build.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * @jest-environment node
+ */
```

Comment:
This was necessary for dart-sass to work and access the file system. If we drop this below our copyright, it doesn't work as expected.

Reply:
I didn't even know this existed. 😂 [Docs](https://jestjs.io/docs/en/configuration#testenvironment-string) show an example of it working below a comment, so it's weird that it doesn't work.

Instead of adding it inline though (where the linter will try to re-add the copyright above it), you _might_ be able to configure it elsewhere, by setting up an override in the [`jest.config.js`](https://github.com/liferay/liferay-frontend-projects/blob/623715898e497fad03bf7cb489fec7537f057bf8/projects/npm-tools/packages/npm-scripts/jest.config.js) for this project (not sure if that will break other tests; it might not seeing as this package is all about node).